page/docs/guide/run.md

```diff
@@ -7,18 +7,19 @@ sidebar_position: 1
 Installing the package adds the `starknet-devnet` command.
 
 ```text
-usage: starknet-devnet [-h] [-v] [--host HOST] [--port PORT] [--load-path LOAD_PATH] [--dump-path DUMP_PATH] [--dump-on DUMP_ON]
+usage: starknet-devnet [-h] [-v] [--verbose] [--host HOST] [--port PORT] [--load-path LOAD_PATH] [--dump-path DUMP_PATH] [--dump-on DUMP_ON]
                        [--lite-mode] [--blocks-on-demand] [--accounts ACCOUNTS] [--initial-balance INITIAL_BALANCE] [--seed SEED]
-                       [--hide-predeployed-accounts] [--start-time START_TIME] [--gas-price GAS_PRICE] [--allow-max-fee-zero]
+                       [--hide-predeployed-contracts] [--start-time START_TIME] [--gas-price GAS_PRICE] [--allow-max-fee-zero]
                        [--timeout TIMEOUT] [--account-class ACCOUNT_CLASS] [--fork-network FORK_NETWORK] [--fork-block FORK_BLOCK]
                        [--fork-retries FORK_RETRIES] [--chain-id CHAIN_ID] [--disable-rpc-request-validation]
-                       [--disable-rpc-response-validation]
+                       [--disable-rpc-response-validation] [--hide-logs]
 
 Run a local instance of Starknet Devnet
 
 optional arguments:
   -h, --help            show this help message and exit
   -v, --version         Print the version
+  --verbose             Show more verbose output. Have higher priority than --hide-logs and --hide-predeployed-contracts
```

Comment:
Have -> Has
This section of `run.md` should be a copy of the output of `starknet-devnet --help`. So any help string you had in mind should first be configured in devnet_config.py

Reply:
Did you apply this copy-paste rule to all help strings?